services/kaze/controllers/application.go

```diff
@@ -228,29 +230,40 @@ func TransferApplicationOwnership(c *gin.Context) {
 
 // FetchMetrics retrieves the metrics of an application's container
 func FetchMetrics(c *gin.Context) {
+	c.Writer.Header().Set("Content-Type", "text/event-stream")
 	appName := c.Param("app")
-	filter := utils.QueryToFilter(c.Request.URL.Query())
-	var timeSpan int64
-
-	for unit, converter := range timeConversionMap {
-		if val, ok := filter[unit].(string); ok {
-			timeVal, err := strconv.ParseInt(val, 10, 64)
-			if err != nil {
-				continue
-			}
-			timeSpan += timeVal * converter
-		}
+	if appName == "" {
+		c.AbortWithStatusJSON(400, gin.H{
+			"success": false,
+			"error":   "App name not provided for metrics",
+		})
```

Comment:
a return statement is required after this to terminate the function

Reply:
Also you can leave the check of `appName == ""`.

This is already handled by the `IsAppOwner` middleware. You cannot create an app with an empty string as name. This rule is already enforced.

Reply:
@supra08 remove the check for `appName == ""`

Reply:
done